src/StoreApi/SessionHandler.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Automattic\WooCommerce\StoreApi;
+
+use Automattic\Jetpack\Constants;
+use Automattic\WooCommerce\StoreApi\Utilities\JsonWebToken;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * SessionHandler class
+ */
+final class SessionHandler extends \WC_Session {
+	/**
+	 * Token from HTTP headers.
+	 *
+	 * @var string
+	 */
+	protected $token;
+
+	/**
+	 * Table name for session data.
+	 *
+	 * @var string Custom session table name
+	 */
+	protected $_table; // phpcs:ignore PSR2.Classes.PropertyDeclaration.Underscore
```

Comment:
I don't think we need to use the _ prefix here? The property is already declared as protected.